docs/dictionaries/heWoedlistReadme.txt

```diff
@@ -0,0 +1,665 @@
+hebrew wordlist is from this project
```

Comment:
There is a spelling mistake in the name of the file. It must be "wordlist" not "woedlist".